### ISSUE TITLE
Tables should have a programmatically-associated caption or name

### ISSUE THREAD

**Issue by axamei** [pulibrary/pul_library_drupal#1472]
If we do not include caption / name, then screen reader users might have a difficult time with pages that have multiple tables, i.e., the screen reader's list of tables will only display the number of columns and rows as the name of the tables. Instead, provide a programmatically-associated caption or name. 

Examples:

```
<caption>1st Quarter Results</caption>
```
```
<table aria-label="Second quarter Results">
```
```
<table>
      <caption><span class="visually-hidden">1st Quarter Results</span></caption>
```
```
<h3 id="tableCaption">Third Quarter Results</h3>
<table aria-labelledby="tableCaption">
```
```
<figure>
<table aria-labelledby="figCaption">
<!-- the rest of the table goes here -->
</table>
<figcaption id="figCaption">Fourth Quarter Results</figcaption>
</figure>
```

Further, the name/caption of a data table SHOULD describe the identity or purpose of the table accurately, meaningfully, and succinctly. And, the name/caption of each data table SHOULD be unique within the context of other tables on the same page.
